src/app/card/card.component.html

```diff
@@ -4,16 +4,16 @@
 
   <div class="card-content {{ textClass }}">
 
-    <span class="card-title">
+    <div class="card-title" *ngIf="hasCardTitle">
```

Comment:
true, sorry I forgot that title is not an `@Input` but a `ng-content` content projection 👍 The getter still a good way to improve the code tho